pages.fr/common/ifconfig.md

```diff
@@ -1,6 +1,7 @@
 # ifconfig
 
 > Configurateur des interfaces réseau.
+> Plus d'informations : <https://manned.org/ifconfig.8>.
```

Comment:
```suggestion
> Plus d'informations: <https://manned.org/ifconfig.8>.
```

Reply:
It seems that most of the information links in **pages.fr** have a space in front of the colon. I'm not sure if it is proper without that space.

Reply:
I had a look at the main `pages`, it may be an innocent typo that a space crept in before a `:`

Reply:
That's not a typo. In french you put a protected space before every colon.
@blueskyson I assume you used the `set-more-info-link.py` script for that?

Reply:
However, you can't add it back via the GitHub web interface, since in converts protected spaces automatically to normal spaces.

Reply:
Got it!